rune/libenclave/attestation/challenger.go

```diff
@@ -2,26 +2,23 @@ package attestation // import "github.com/inclavare-containers/rune/libenclave/a
 
 import (
 	"fmt"
+	"github.com/inclavare-containers/rune/libenclave/attestation/registration"
+	_ "github.com/inclavare-containers/rune/libenclave/attestation/sgx/challenger"
+	"github.com/inclavare-containers/rune/libenclave/attestation/utils"
 )
 
 type Challenger interface {
 	Name() string
 	New(map[string]string) error
 	Check([]byte) error
-	Verify([]byte) (*ReportStatus, error)
-	GetReport([]byte, uint64) (*ReportStatus, map[string]string, error)
-	ShowReportStatus(*ReportStatus)
+	Verify([]byte) (*utils.ReportStatus, error)
+	GetReport([]byte, uint64) (*utils.ReportStatus, map[string]string, error)
+	ShowReportStatus(*utils.ReportStatus)
```

Comment:
ReportStatus should reside in interface layer, otherwise the application has to explicitly import attestation/utils. This also means the implementation layer should not know ReportStatus, so he implementation layer just returns the status information with the basic data type in golang.